src/main/resources/db/migration/refdata/V1_7__upw_assessment.sql

```diff
@@ -216,6 +217,7 @@ VALUES
 ('19a1b4d8-0f85-4049-ba7b-127646c882d8', 'pregnancy_recently_given_birth_details', '2021-09-27 14:50:00', null, 'textarea', null, 'Give details', null, null),
 ('155e1596-9f83-4e0a-8fa7-e36f9ed2b2e5', 'health_issues_complete', '2021-09-27 14:50:00', null, 'radio', '8067ff6e-7400-4d1e-ae2a-87dee7e124ec', 'Mark this section as complete?', '', null),
 ('0ba4ef3e-8c40-4c1e-b267-a42afd07ee4e', 'gp_details_complete', '2021-09-27 14:50:00', null, 'radio', '8067ff6e-7400-4d1e-ae2a-87dee7e124ec', 'Mark this section as complete?', '', null),
+('2a1a41dd-5d77-4674-9f8f-561c654867b8', 'gp_details', '2021-09-27 14:50:00', null, 'table', null, '', '', null),
```

Comment:
Should we be changing existing flyway SQL files?

Reply:
Normally, definitely not but this has been discussed and given a thumbs up in this instance as it's reference data and separate from the user data